api/justfile

```diff
@@ -160,6 +160,13 @@ generate-docs doc="media-props" fail_on_diff="true":
 test *args: wait-up
     env DC_USER="ov_user" just ../exec web pytest "$@"
 
+# When the unit tests run concurrently with the integration tests, the integration tests fail
+# because the unit tests drop the real database in the end. This is a workaround to run the unit
+# tests first and then the integration tests.
```

Comment:
Small nit: a single-line comment here would be preferable as this will be shown when running `just` with no arguments. The long one can be inside the recipe.